packages/generator/new-action/src/run.ts

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env node
+
+import { Command } from '@modern-js/utils/commander';
+
+import { MWANewAction } from './mwa';
+import { ModuleNewAction } from './module';
+import { MonorepoNewAction } from './monorepo';
+
+import { getSolutionByDependance } from './utils';
+
+const main = async () => {
+  const program = new Command();
+  program
+    .option('--solution <solution>', 'solution', '')
+    .option('--config <config>', 'config', '{}')
+    .option('--root-path <rootPath>', 'project root path', '')
+    .action(async params => {
+      if (!params.solution) {
```

Comment:
It looks like a breaking change. Will the `new` command for `module-tools` templates append `--solution` ?

Reply:
I see..no problem